### ISSUE TITLE
Route.URL() does not include query string described by Queries()

### ISSUE THREAD

**Issue by ChrisHines** [gorilla/mux#255]
To reproduce:

```go
r := mux.NewRouter()
route := r.Path("/path/{var1}").Queries("q1", "{q1}", "q2", "{q2}")
u, _ := route.URL("var1", "1", "q1", "2", "q2", "3")
fmt.Println(u)
```

Output: `/path/1`
Expected: `/path/1?q1=2&q2=3`

**Comment by ChrisHines** [comment]
It appears #230 is already in progress to address this shortcoming.

**Comment by kisielk** [comment]
Implemented in #270
